src/app/pages/storage/import-disk/import-disk.component.ts

```diff
@@ -180,8 +180,8 @@ export class ImportDiskComponent implements OnDestroy, Formconfiguration {
     this.dialogRef.componentInstance.submit();
     this.dialogRef.componentInstance.success.subscribe((job_res) => {
```

Comment:
Why are we reaching this code if the job didn't succeed?

Reply:
From what I gather, ABORTED is a valid state for the job and not failure. Failure is when there's an error in job execution. ABORTED is a valid state where no error occurred. We could improve the code by make a separate clearer indication like `dialogRef.componentInstance.success` (maybe `dialogRef.componentInstance.aborted`)

Reply:
Yes, lets do that, please. Treating ABORTED as success would be wrong.

Reply:
The option to abort isn't provided in the UI, however. At least, not in this case. This case occurs when the job is initiated from the UI but aborted from the machine shell directly.

Reply:
Right, but soon enough we will let it be done via UI